now.json

```
@@ -12,10 +12,18 @@
   },
   "builds": [
     {
-      "src": "rasterize.js",
+      "src": "index.js",
       "use": "@now/node",
       "config": { "maxLambdaSize": "40mb" }
     }
   ],
-  "routes": [{ "src": "/(.*)", "dest": "/rasterize.js" }]
+  "routes": [
+    { "src": "/(.*)", "dest": "/index.js" },
```

```suggestion
    { "src": "/(.+)", "dest": "/index.js" },
```

The order here matters so you should move this below the root redirect `/`. Also the regex can be changed to help